Recrutify/Recrutify.Host/Configuration/Profiles/CandidateProfile.cs

```diff
@@ -13,7 +12,7 @@ public CandidateProfile()
             CreateMap<CandidateCreateDTO, Candidate>()
                 .ForMember(x => x.ProjectResults, opt => opt.Ignore())
```

Comment:
rename all x to dest, cause somewhere is dest somewhere is x

Reply:
done